leetcode/242.ValidAnagram.cpp

```diff
@@ -0,0 +1,53 @@
+class Solution
+{
+public:
+    // First case using a dictionary
+    bool isAnagram1(string s, string t)
```

Comment:
consider using std::string_view if you only need a read-only string

Reply:
addressed